src/types.ts

```diff
@@ -14,6 +14,8 @@ export enum OutgoingActionType {
   TURN_OFF_NAMES = 'turn_off_names',
   TURN_ON_SEVERAL_TRANSCRIPTS = 'turn_on_several_transcripts',
   TURN_OFF_SEVERAL_TRANSCRIPTS = 'turn_off_several_transcripts',
+  TURN_ON_TRANSCRIPTS = 'turn_on_transcripts',
+  TURN_OFF_TRANSCRIPTS = 'turn_off_transcripts',
```

Comment:
Why do we have this pair of actions again? I thought the idea is that you are always telling the genome browser the list of transcript ids that you want to be visible, in which case the "on" and "off" actions are identical.